### PR TITLE
Latest updates for JWebbinar 2 sessions

### PR DESCRIPTION
**Pull request by cslocum**
Just minor changes.  I want to get these in so I can tag this state.

### CODE REVIEW

File: deployments/jwebbinar/config/dev.yaml [hunk 1]
```diff
@@ -3,3 +3,13 @@ jupyterhub:
     continuous:
       # Don't need continuous pre-puller on staging
       enabled: true
+  singleuser:
```
Comment (cslocum) [inline deployments/jwebbinar/config/dev.yaml:6]: @jmatuskey - these should no longer be here, right?  Weren't they moved somewhere more general?

File: deployments/jwebbinar/image/environments/frozen/jdaviz-frozen.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: jdaviz
+^[[Aname: jdaviz
```
[the remark this answers is not in the document]
Comment (cslocum) [reply deployments/jwebbinar/image/environments/frozen/jdaviz-frozen.yml:1]: Oh Yikes!  Glad you caught that.

File: deployments/jwebbinar/image/Dockerfile.custom [hunk 3]
```diff
@@ -90,15 +93,21 @@ USER root
 
 # Install additional lab and notebook extensions
 USER ${NB_UID}
-RUN jupyter nbextension install --py --sys-prefix widgetsnbextension && \
+
```
[the remark this answers is not in the document]
Comment (jaytmiller) [reply deployments/jwebbinar/image/Dockerfile.custom:96]: OK.  Think the Dockerfile.custom version can be added to Dockerfile.trailer (later maybe)?   If that order is OK that would solve the issue for all missions.   Another thing to do might be to factor the setup out into something like common-scripts/setup-lab-extensions and then just RUN it wherever needed.   Again,  maybe later.

**Review by jaytmiller — APPROVED** [review]
Some ideas about refactoring lab extension setup later,  but LGTM!

File: deployments/jwebbinar/image/Dockerfile.custom (hunk 3 above)
[the remark this answers is not in the document]
Comment (cslocum) [reply deployments/jwebbinar/image/Dockerfile.custom:96]: Those are both good ideas.  Let me pick this back up tomorrow.

**Pull request merged**